vue/src/store.ts

```diff
@@ -58,10 +68,22 @@ export const state = reactive<{
     satelliteBounds: [],
     imageOpacity: 0.75,
     loadingSatelliteImages: false,
+    satelliteTimeSource: 'S2',
+    cloudCover: 100,
+    satelliteSources: ['S2'],
```

Comment:
Is worldview not included here because we're still having issues with loading tiles?

Reply:
Yeah, I don't want it to default to on, I only want people turning it on if they know what they're doing.